pkgs/os-specific/linux/net-tools/default.nix

```diff
@@ -13,7 +13,7 @@ stdenv.mkDerivation rec {
       cp ${./config.h} config.h
     '';
 
-  makeFlags = "BASEDIR=$(out) mandir=/share/man";
+  makeFlags = "BASEDIR=$(out) mandir=/share/man HAVE_ARP_TOOLS=1 HAVE_PLIP_TOOLS=1 HAVE_SERIAL_TOOLS=1 HAVE_HOSTNAME_TOOLS=1";
```

Comment:
I think it might be better to split the `makeFlags` into an array as this might be more readable.